_quarto.yml

```diff
@@ -18,9 +18,12 @@ website:
           - list-collections.ipynb
         - section: "Advanced API Use"
           contents:
+          ## disable the 2 following notebooks since they rely on data 
+          ## in MCP and github actions has no way to request short term
+          ## access credentials to access the data
```

Comment:
I don't understand this comment in this place, I understand why we want to disable them but what is disabling them here?

Reply:
That is absolutely correct. I forgot to remove the comment once I figured out how to use the `freeze` option to avoid re-rending the notebooks!

Reply:
Great catch! Thanks!